docs/source/quickstart.rst

```diff
@@ -42,31 +42,20 @@ For Deep learning research
+`PyTorch Lightning <https://github.com/PyTorchLightning/pytorch-lightning>`_ is designed to abstract away unnecessary boilerplate, while enabling maximal flexibility. In order to provide full flexibility, solving very common deep learning problems such as classification in Lightning still requires some boilerplate. It can still take quite some time to get a baseline model running on a new dataset or out of domain task. We created Flash to answer our users need for a super quick way to baseline for Lightning using proven backbones for common data patterns. Flash aims to be the easiest starting point for your research- start with a Flash Task to benchmark against, and override any part of flash with Lightning or PyTorch components on your way to SOTA research.
 
 Flexibility where you want it
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Flash tasks are essentialy LightningModules, and the Flash Trainer is a thin wrapper for the Lightning Trainer. You can use your own LightningModule instead of the Flash task, the Lightning Trainer instead of the flash trainer, etc. Flash helps you focus even more only on your research, and less on anything else.
+Flash tasks are essentially LightningModules, and the Flash Trainer is a thin wrapper for the Lightning Trainer. You can use your own LightningModule instead of the Flash task, the Lightning Trainer instead of the flash trainer, etc. Flash helps you focus even more only on your research, and less on anything else.
 
 Standard best practices
 ^^^^^^^^^^^^^^^^^^^^^^^
-Flash tasks implement the standard best practices for a variety of diffrent models and domains, to save you time digging through different implementations. Flash abstracts even more details than lightning, allowing deep learning experts to share their tips and tricks for solving scoped deep learning problems.
+Flash tasks implement the standard best practices for a variety of different models and domains, to save you time digging through different implementations. Flash abstracts even more details than Lightning, allowing deep learning experts to share their tips and tricks for solving scoped deep learning problems.
 
 .. tip::
 
     Read :doc:`here <reference/flash_to_pl>` to understand when to use Flash vs Lightning.
 
-----
-
-Install
```

Comment:
why do you remove this ?

Reply:
Because we already have the installation instruction at https://lightning-flash.readthedocs.io/en/latest/installation.html. Should we at least paste the link here instead of removing it?

Reply:
sure - it's fine then :)